components/accordion/docs.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
This is taken from package.json. We should only have one source of truth and this PR should also get rid of what's defined in package.json.

Reply:
As we spoke, this will have to be a two-step process. I will leave package.json untouched and modify the doc site repo to use individual `docs.json` in `design-system-react-site/scripts/docs.js` script. Then, I'll create a new PR to remove the components from package.json.

Reply:
That sounds good and a great idea for allowing releases! `brand-band` will most likely be merged in, in the mean time.